### PR TITLE
#159951819 Users should be able to login

### PR DESCRIPTION
**Pull request by abulojoshua1**
### What does this PR do?
- Adds login functionality to authors haven front end

### Description of Task to be completed?
- Make the users able to log in on authors haven using an interface

### How should this be manually tested?
- Clone the application and run NPM start
- Click the login button in the top right corner of the screen and log in

### Any background context you want to provide?
- I created a login component and added it to the base UI
### What are the relevant pivotal tracker stories?
- [#159951819](https://www.pivotaltracker.com/story/show/159951819)

### CODE REVIEW

**Review by lytes20 — APPROVED** [review]

**Review by phillipseryazi — APPROVED** [review]

File: src/components/footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (geofrocker) [inline src/components/footer/Footer.js:1]: Was this part of your work?

File: src/tests/components/Alert.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { shallow } from 'enzyme';
+import Alerts from '../../components/Alerts/Alerts';
+
+describe('<Alerts />', () => {
+  const props = {
+    alertType: 'alert alert-danger', title: 'Error', message: 'some text',
+  };
+  it('should render a bootstrap alert', () => {
+    const wrapper = shallow(<Alerts {...props} />);
+    // const wrapperStyle = wrapper.get(0).props.style;
```
Comment (geofrocker) [inline src/tests/components/Alert.test.js]: Do you need this?

File: src/tests/reducers/loginReducer.test.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+describe('toggleLandingPageReducer', () => {
+  it('should return the initial state', () => {
+    expect(authReducer(undefined, {})).toEqual({
+      isAuthentic: false,
+      loading: false,
+      toggleProfileDropdown: false,
+    });
+  });
+
+  it('enabling the profile picture dropdown', () => {
```
Comment (geofrocker) [inline src/tests/reducers/loginReducer.test.js]: Can you make this more readable

File: src/tests/components/Alert.test.js (hunk 2 above)
Comment (abulojoshua1) [reply src/tests/components/Alert.test.js]: I will remove that asap

File: src/tests/reducers/loginReducer.test.js (hunk 3 above)
Comment (abulojoshua1) [reply src/tests/reducers/loginReducer.test.js]: Ok, I will re-phrase that too.

File: src/components/footer/Footer.js (hunk 1 above)
Comment (abulojoshua1) [reply src/components/footer/Footer.js:1]: Since I had started working on the base UI before @AGMETEOR took on that task. I also had that file in my code.

**Pull request merged**